<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="dashboardTab">
     <attribute name="title">
      <string>Dashboard</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="budgetTab">
     <attribute name="title">
      <string>Budget</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="expensesTab">
     <attribute name="title">
      <string>Expenses</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="incomeTab">
     <attribute name="title">
      <string>Income</string>
     </attribute>
     <widget class="QTableView" name="incomeTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addIncomeButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveIncomeButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteIncomeButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loansTab">
     <attribute name="title">
      <string>Loans</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="assetsTab">
     <attribute name="title">
      <string>Assets</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="transactionsTab">
     <attribute name="title">
      <string>Transactions</string>
     </attribute>
     <widget class="QTableView" name="transactionsTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>441</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="saveTransactionButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
